descope/client/descope.go

```diff
@@ -0,0 +1,62 @@
+package client
```

Comment:
I am wondering if we should name this file `client.go`, it is already in `descope` folder (sub package)